### ISSUE TITLE
no-unused-modules should honor exports field

### ISSUE THREAD

**Issue by httpete** [import-js/eslint-plugin-import#2525]
we use :

```
exports: './index.ts'

```
In our package.json, and clearly anything exported from this file should not be flagged as unused.

**Comment by JounQin** [comment]
duplicate of #2132
